src/autoscaler.py

```diff
@@ -75,6 +87,7 @@ def authorize(self, container):
         container.push(*self.accounts_file, make_dirs=True, **self.root_owned)
         container.push(*self.controller_file, make_dirs=True, **self.root_owned)
         container.push(*self.models_file, make_dirs=True, **self.root_owned)
```

Comment:
i'd expect you'd want to remove these @stonepreston?

Reply:
yes if we get HA working with the [simple connector PR](https://github.com/charmed-kubernetes/autoscaler/pull/6) then we would no longer need to build up these client side files and push them into the container